<launch>
  <arg name="DEV" default="/dev/input/mouse0" />
  <arg name="JOINT_STATES" default="/joint_states" />
  <arg name="JOINT_TRAJECTORY_ACTION" default="/head_traj_controller/follow_joint_trajectory" />
  <arg name="PITCH_JOINT" default="head_pan_joint" />
  <arg name="YAW_JOINT" default="head_tilt_joint" />

  <include file="$(find joy_mouse)/launch/kensington_trackball.launch">
    <arg name="AUTOREPEAT_RATE" value="30.0" />
    <arg name="DEV" value="$(arg DEV)" />
  </include>
  <node pkg="jsk_teleop_joy"
        type="head_control_by_trackball.py"
        name="head_trackball_control"
        output="screen">
    <remap from="/joint_states" to="$(arg JOINT_STATES)"/>
    <param name="/joint_trajectory_action" value="$(arg JOINT_TRAJECTORY_ACTION)" />
    <param name="/pitch_joint" value="$(arg PITCH_JOINT)" />
    <param name="/yaw_joint" value="$(arg YAW_JOINT)" />
  </node>
</launch>
